miden-crypto/src/merkle/mmr/tests.rs

```diff
@@ -12,6 +12,23 @@ use crate::{
     merkle::{InOrderIndex, MerklePath, MerkleTree, MmrProof, NodeIndex, int_to_node},
 };
 
+#[test]
+fn tests_empty_mmr_peaks() {
+    let peaks = MmrPeaks::empty();
+    assert_eq!(peaks.num_peaks(), 0);
+    assert_eq!(peaks.num_leaves(), 0);
+}
```

Comment:
Another test could be to construct an empty `Mmr` and then get peaks from it and make sure these are equal to empty `MmrPeaks`.

Reply:
The test below does that - unless I'm misunderstanding your comment:
```rust
fn test_empty_partial_mmr() {
    let mmr = PartialMmr::empty();
    assert_eq!(mmr.num_leaves(), 0);
    assert_eq!(mmr.forest(), 0);
    assert_eq!(mmr.peaks(), MmrPeaks::empty());
```

Reply:
Ah - sorry - missed it!